vagrant/provisioning/roles/alfresco/files/passencode.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+
+import bcrypt
```

Comment:
with using the web script, we don't need the bcrypt changes any more, and we don't need the database update. The database update didn't work for me anyway, at least not on PostgreSQL.